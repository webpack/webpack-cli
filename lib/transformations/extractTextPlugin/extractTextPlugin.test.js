const defineTest = require('../defineTest');

defineTest(__dirname, 'extractTextPlugin');
