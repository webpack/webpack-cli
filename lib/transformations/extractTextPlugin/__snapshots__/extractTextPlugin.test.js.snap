exports[`extractTextPlugin transforms correctly 1`] = `
"let ExtractTextPlugin = require(\'extract-text-webpack-plugin\');

module.export = {
    module: {
        rules: [
            {
                test: /\\.css$/,
                use: ExtractTextPlugin.extract({
                    \'fallback\': \'style-loader\',
                    \'use\': \'css-loader\'
                })
            }
        ]
    },
    plugins: [
        new ExtractTextPlugin(\"styles.css\"),
    ]
}
"
`;
